src/SFA.DAS.EmployerAccounts.Events/Messages/ApprenticeChangesRequestedMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+
+namespace NuGet.Messages
+{
+    public class ApprenticeChangesRequestedMessage: Message
```

Comment:
These messages are in the commitments project. They should not be added in this project as they are not related to MA. I saw Nuget in the namespace and alarm bells started to ring. You'll need to change those messages in the commitments project.

Reply:
They are not in the solution. The files are on disk and not visible in VS. I'll delete them from disc.

Reply:
They were test files I used to make a realistic demo.

Reply:
ok, that's fine. They just don't need to be committed that's all.